packages/block-editor/src/store/selectors.js

```diff
@@ -533,7 +535,12 @@ export const getBlockParentsByBlockName = createSelector(
 					id,
 					name: getBlockName( state, id ),
 				} ) ),
-				{ name: blockName }
+				( { name } ) => {
+					if ( Array.isArray( blockName ) ) {
+						return blockName.includes( name );
+					}
+					return name === blockName;
+				}
```

Comment:
Stealing all this from @noahtallen 's nice addition on #25547 that never went through for other reasons.  Its definitely helpful. 😁

Reply:
what is this change about?

Reply:
Let's a unit test to this selector to cover that

Reply:
updated